dts/bindings/sensor/hzgrow,grow-r502a.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021, Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+        HZ-Grow GROW_R502A Fingerprint sensor.
+
+compatible: "hzgrow,grow-r502a"
```

Comment:
Make this `hzgrow,r502a` (Update filename to match)

Reply:
Then renaming from grow_r502a to r502a in all the file names also makes sense right?

Reply:
Any where that you had duplicate HZGROW, GROW R502a.  Its the duplicate that needs cleaning up.

Reply:
Okay, Fixed!